messages/de/app.php

```diff
@@ -131,12 +131,12 @@
     'Select a Category' => 'Kategorie auswählen',
     'Select Language' => 'Sprache auswählen',
     'Select a Server' => 'Server auswählen',
-    'Select a server to add to the public server list on DISBOARD' => 'Wähle einen Server aus, der zur öffentlichen Serverliste auf Disboard hinzugefügt werden soll',
+    'Select a server to add to the public server list on DISBOARD' => 'Wähle einen Server aus, um ihn zur öffentlichen Serverliste von Disboard hinzuzufügen',
```

Comment:
The original translation corresponds more to the original text. DISBOARD should be all upper case, though, and that's wrong in the original translation.